include/tallies/MeshTally.h

```diff
@@ -113,4 +113,9 @@ class MeshTally : public TallyBase
   std::unique_ptr<libMesh::ReplicatedMesh> _libmesh_mesh_copy;
   /// A mapping between the OpenMC bins (active block restricted elements) and all elements.
   std::vector<unsigned int> _bin_to_element_mapping;
-};
+  /// extra_integer_id_name
+  std::string _extra_integer_name;
+  unsigned int _extra_integer_index;
```

Comment:
Let's refer to these using the term `cluster` somehow so we know what the extra integers are for.
```suggestion
  /// extra_integer info for clustering
  std::string _clustering_name;
  unsigned int _clustering_index;
```